rest/taskana-rest-spring/src/main/java/pro/taskana/rest/resource/WorkbasketSummaryResource.java

```diff
@@ -37,6 +37,7 @@
     public String orgLevel2;
     public String orgLevel3;
     public String orgLevel4;
+    private Boolean markedForDeletion;
```

Comment:
didn't we want to change it to boolean to make the BeanCopy work?

Reply:
of course!
Done